### ISSUE TITLE
The empty DataGridView control cannot get a blue Narrator rectangle, just get the focus dot line

### ISSUE THREAD

**Issue by Vino-Wang** [dotnet/winforms#2178]
* .NET Core Version: (.NET Core SDK 3.1 Preview 2 3.1.100.14533)
* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
The empty DataGridView control cannot get a blue Narrator rectangle, just get the focus dot line, so no any announcement about the empty dataGridView.
But when adding some columns to the DataGridView, the blue rectangle will on the currently specific cells with corresponding announcement by Narrator reading. Maybe this issue is by design, just report here and find the final confirmation from you guys.

This issue also can reproduce with latest .NET Framework 4.8 project.

**Actual behavior:** 
The empty DataGridView control cannot get a blue Narrator rectangle, just get the focus dot line.

**Expected behavior:**
The empty DataGridView control should get a blue Narrator rectangle.

**Minimal repro:**
1.	Run the attached app,
2.	Launch the Narrator and tab to navigate to the controls on the form.
3.	Observe the narrator blue rectangle between controls and DataGridView.
[WindowsFormsApp3.zip](https://github.com/dotnet/winforms/files/3766270/WindowsFormsApp3.zip)

**Comment by merriemcgaw** [comment]
@Vino-Wang is this still happening? Is Narrator reading the right stuff?

**Comment by Zheng-Li01** [comment]
@merriemcgaw verified the issue with latest .NET Core SDK 3.1[3.1.200.14989] build, the issue still can preproduce.

**Comment by merriemcgaw** [comment]
@Zheng-Li01 does Narrator read it properly and it just doesn't have the blue indicator, or are you saying you can't get Narrator to recognize it has focus at all?

**Comment by Zheng-Li01** [comment]
@merriemcgaw, the empty DGV can't get Narrator to recognize it has focus at all. Please see below reproduce gif.
![core](https://user-images.githubusercontent.com/38325459/75226530-762ddb00-57e7-11ea-917c-02adf6ca3285.gif)

**Comment by merriemcgaw** [comment]
Thanks @Zheng-Li01 I think we will leave this marked as future. There isn't any real user impact as this is the empty datagrid view, and that's not a common scenario for customers.

**Comment by Lydia-Shi** [comment]
Verified this issue with .NET 7.0.100-alpha.1.21566.13 from latest main branch, this issue is fixed that empty DGV can get blue Narrator rectangle.
![image](https://user-images.githubusercontent.com/54019172/142148816-672d66bc-88a4-40af-89ca-8c7319f1c2e6.png)
